### PR TITLE
Adding support to provide custom labels to sequence classification

### PR DESCRIPTION
**Pull request by lalitpagaria**
To complete https://github.com/cdpierse/transformers-interpret/issues/25

### CODE REVIEW

**Comment by lalitpagaria** [comment]
@cdpierse Can you please review PR

File: transformers_interpret/explainers/sequence_classification.py [hunk 1]
```diff
@@ -50,12 +48,14 @@ def __init__(
...
         tokenizer: PreTrainedTokenizer,
         attribution_type: str = "lig",
+        custom_labels: Optional[List[str]] = None
     ):
         """
         Args:
             model (PreTrainedModel): Pretrained huggingface Sequence Classification model.
             tokenizer (PreTrainedTokenizer): Pretrained huggingface tokenizer
             attribution_type (str, optional): The attribution method to calculate on. Defaults to "lig".
+            custom_labels (List[str], optional): To use custom labels instead Pretrained Model configured
```
Comment (cdpierse) [inline transformers_interpret/explainers/sequence_classification.py:58]: I'd like to expand a little bit here on the description of what this does. I'm thinking 

`custom_labels (List[str], optional):  Applies custom labels to label2id and id2label configs. Labels must be same length as the base model configs' labels. Labels and ids are applied index-wise. Defaults to None.`

File: test/test_sequence_classification_explainer.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def test_sequence_classification_explainer_init_bert():
     assert seq_explainer.attribution_type == "lig"
     assert seq_explainer.label2id == BERT_MODEL.config.label2id
     assert seq_explainer.id2label == BERT_MODEL.config.id2label
-    assert seq_explainer.attributions == None
+    assert seq_explainer.attributions is None
```
Comment (cdpierse) [inline test/test_sequence_classification_explainer.py:40]: Nice catch! Thanks.

**Review by cdpierse — COMMENTED** [review]
@lalitpagaria thank you very much for this PR, it's a great addition to the classification explainer and one I've wanted to add for a while, so I really appreciate it. 

I've left one request for a minor change to be made to the docstring but besides that the rest looks good. The implementation is really clean. 

I'm also currently using the black formatter for the entire repo so once you have the next change done could you run `black .` from the project's root. 

Thanks also for writing the additional tests, it's a big help in getting things merged quicker.

**Comment by lalitpagaria** [comment]
Thanks for the review. Updated PR with suggested changes ie doc string and black formatting.

**Comment by cdpierse** [comment]
Amazing work. This is a fantastic PR thanks @lalitpagaria. I'm merging this into dev now and this will be part of the next release 0.4.0 which should hopefully be within the next 2 weeks.

**Comment by lalitpagaria** [comment]
Thanks @cdpierse for this amazing tool
I want to integrate use it with https://github.com/lalitpagaria/obsei
So user able to interpret prediction made by AI models like zero shot, sentiment or NER etc.

**Pull request merged**